contracts/adapters/Onboarding.sol

```diff
@@ -82,15 +83,16 @@ contract OnboardingContract is
     function _submitMembershipProposal(
         DaoRegistry dao,
         address tokenToMint,
-        address applicant,
+        address payable applicant,
+        address payable proposer,
         uint256 value,
         address token
     ) internal returns (uint256) {
         OnboardingConfig storage config = configs[address(dao)][tokenToMint];
         require(config.chunkSize > 0, "config missing");
 
         uint256 numberOfChunks = value.div(config.chunkSize);
-        require(numberOfChunks > 0, "not sufficient funds");
+        require(numberOfChunks > 0, "not sufficient funds"); /* explain why this should not be < instead? */
```

Comment:
we need to have at least 1 numberOfChunks. If numberOfChunks == 0 => not enough funds sent